qiskit/synthesis/evolution/suzuki_trotter.py

```diff
@@ -103,7 +102,9 @@ def synthesize(self, evolution):
     @staticmethod
     def _recurse(order, time, pauli_list):
         if order < 1:
-            raise ValueError("This bitch empty -- yeet!")
+            raise ValueError(
+                "The order of the Trotterization should be an integer number at least " "1."
```

Comment:
We might include feedback with what value was actually provided.

Reply:
Included the value of the given order in the message.

Reply:
I think this whole check can be deleted, I had that for internal checks only and forgot to remove it (as one might notice from the not very helpful error message.. 😄)